src/helpers/User/User.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * User helper.
+ * @module helpers/AuthToken
+ */
+
+/**
+ * Is admin user method.
+ * @method isAdminUser
+ * @returns {undefined}
+ */
+export function isAdminUser(user) {
+  return user?.roles
+    ? user.roles.includes('Site Administrator') ||
+        user.roles.includes('Manager')
+    : false;
+}
```

Comment:
```suggestion
import { intersection } from 'lodash/intersection';

/**
 * Checks if the user has one or more given roles.
 * @method userHasRoles
 * @returns {boolean}
 */
export function userHasRoles(user, roles = []) {
  return intersection(user?.roles ?? [], roles).length > 0;
}
```

How about a slightly different and more generic approach, like this for example? I can't decide whether I would prefer having a direct ``isAdminUser`` method or a more generic one, or maybe both.